digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/AssetService.scala

```diff
@@ -13,6 +13,24 @@ trait AssetOperations {
       assetDao.getMunicipalityById(id)
     }
   }
+
+  def getVerifiableAssets: Seq[String] = {
+    withDynTransaction {
+      assetDao.getVerifiableAssetTypes
+    }
+  }
+
+  //this should be moved to municipalityDAO
+  def getMunicipalitiesNameByCode(municipalityCodes: Set[Int]): (List[Map[String, Any]]) = {
+    val municipalities = withDynTransaction {
+      assetDao.getMunicipalitiesNameByCode(municipalityCodes)
+    }
```

Comment:
follow the comment, move to municipalitydao

Reply:
it will only be possible after merge with master